### PR TITLE
VIH-8196 Hide Adding staff member with feature flags

### PR DESCRIPTION
**Pull request by PrasannaDommalapati**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-8196

### Change description ###

Hide Adding staff member with feature flags

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/assign-judge/assign-judge.component.ts [hunk 1]
```diff
@@ -87,6 +90,7 @@ export class AssignJudgeComponent extends BookingBaseComponent implements OnInit
 
     ngOnInit() {
         this.failedSubmission = false;
+        this.featureService.getFeatureToggles().subscribe(t => (this.showStaffMemberFeature = t.staff_member));
```
Comment (omio-ma) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/booking/assign-judge/assign-judge.component.ts:93]: needs to be unsubscribed when this component gets destroyed, or if we only need to call this once we can use the first or take operators from rxjs

**Review by ZJianWang — APPROVED** [review]
LGTM

File: AdminWebsite/AdminWebsite/ClientApp/src/app/services/feature-toggle.service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    });
+
+    it('should be created', () => {
+        expect(service).toBeTruthy();
+    });
+    it('should return the feature config for staff member feature turned on', () => {
+        featureConfig.staff_member = true;
+        clientApiSpy.getFeatureToggles.and.returnValue(of(featureConfig));
+
+        service.getFeatureToggles().subscribe(config => {
```
Comment (omio-ma) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/services/feature-toggle.service.spec.ts:30]: this is interesting as we are subscribing to an observable in a test so we might have to unsubscribe from here as well, I would suggest however either to use async function here and await or use fakeasync instead.

File: AdminWebsite/AdminWebsite.UnitTests/Controllers/FeatureControllerTests.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+            _mocker = AutoMock.GetLoose();
+            _mocker.Mock<IOptions<FeatureToggleConfiguration>>().Setup(opt => opt.Value).Returns(new FeatureToggleConfiguration()
+            {
+                StaffMember = true
+            });
+
+            var _controller = _mocker.Create<FeatureController>();
+            var result = _controller.GetFeatureToggles();
+
+            result.Value.StaffMember.Should().Be(true);
```
Comment (omio-ma) [inline AdminWebsite/AdminWebsite.UnitTests/Controllers/FeatureControllerTests.cs:27]: you can use .Should().BeTrue() here as the value here will be boolean

**Review by ZJianWang — APPROVED** [review]

**Review by kevin-hmcts — APPROVED** [review]
LGTM

**Pull request merged**